sdk/core/azure-core/azure/core/settings.py

```diff
@@ -153,7 +165,7 @@ def convert_tracing_impl(value: Union[str, Type[AbstractSpan]]) -> Optional[Type
 
     """
     if value is None:
-        return get_opencensus_span_if_opencensus_is_imported()
+        return get_opentelemetry_span_if_opentelemetry_is_imported()
```

Comment:
Is it by design to change the default behavior?

Reply:
Yea, since the opentelemetry plugin should be the preferred option going forward. However, it doesn't look like we ever actually pass in a `None` value for `value` in all the places we call this function.. So, in actuality, there shouldn't be any behavioral changes.

Reply:
Interesting. You mean this is dead code? :)

Reply:
I think for the most part, but I think it still might be worth keeping as a fallback just in case. Conceviably, [here](https://github.com/Azure/azure-sdk-for-python/blob/4da92f52700e793a38203062094b951d73a7e1a4/sdk/core/azure-core/azure/core/settings.py#L251) could potentially pass `None`, but I'm not sure how system hooks are used in this context.

Reply:
If open telemetry is not installed but open census is, should we return get_opencensus_span_if_opencensus_is_imported?

Reply:
Makes sense, I can have it check both here.